Terminal.Gui/Views/TextField.cs

```diff
@@ -755,7 +784,10 @@ void RedoChanges ()
 			//Adjust ();
 		}
 
-		void UndoChanges ()
+		/// <summary>
+		/// Undoes the latest changes.
+		/// </summary>
+		public void Undo ()
 		{
 			if (ReadOnly)
 				return;
```

Comment:
Missing braces

Reply:
Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/f9846b42bd137c6b806323b6b26258e22b9ff55c.